package.json

```diff
@@ -85,15 +85,15 @@
   "homepage": "https://github.com/thepracticaldev/dev.to#readme",
   "devDependencies": {
     "@stoplight/spectral": "5.7.1",
-    "@storybook/addon-a11y": "^6.1.3",
-    "@storybook/addon-actions": "^6.0.28",
-    "@storybook/addon-knobs": "^6.1.4",
-    "@storybook/addon-links": "^6.1.3",
+    "@storybook/addon-a11y": "^6.1.5",
```

Comment:
I believe @rhymes mentioned that Dependabot can group updates. I wonder if we can somehow configure it to batch update Storybook dependencies?

Reply:
I don't remember if that's already possible or if they were working on it - @maestromac do you remember?

Reply:
I remember @rhymes pointed out https://github.com/forem/forem/pull/10447. I haven't seen it being applied to storybook though

Reply:
Maybe it works for Ruby but not yet for JavaScript?